src/slack_annotations/cli.py

```diff
@@ -24,4 +25,10 @@ def cli(argv=None):
     else:
         search_params = None
 
-    print(notify(search_params, args.token, args.cache_path))
+    annotations = notify(
+        search_params=search_params,
+        token=args.token,
+        cache_path=args.cache_path,
+        group_name=args.group_name,
+    )
+    print(json.dumps(annotations))
```

Comment:
Made it so `notify` returns dict and only then we convert that into a string for simplicity and ease of testing.